heron/packing/src/java/com/twitter/heron/packing/roundrobin/RoundRobinPacking.java

```diff
@@ -144,11 +143,11 @@ public PackingPlan pack() {
           new PackingPlan.ContainerPlan(containerId, instancePlanMap, resource);
 
       containerPlanMap.put(containerId, containerPlan);
+      topologyRam += containerRam;
```

Comment:
KK, found the line which requesting one more container for TMaster. Could you add one line comment for it?